miqa/core/models/scan.py

```diff
@@ -5,21 +5,22 @@
 from django.db import models
 from django_extensions.db.models import TimeStampedModel
 
-SCAN_TYPES = [('ncanda-t1spgr-v1', 'ncanda-t1spgr-v1'), ('ncanda-mprage-v1', 'ncanda-mprage-v1')]
+SCAN_TYPES = [
+    ('T1', 'T1'),
+    ('T2', 'T2'),
+    ('FMRI', 'FMRI'),
+    ('MRA', 'MRA'),
+    ('PD', 'PD'),
+    ('DTI', 'DTI'),
+    ('DWI', 'DWI'),
+]
 
 
 class Scan(TimeStampedModel, models.Model):
     class Meta:
-        constraints = [
-            models.UniqueConstraint(
-                fields=['experiment', 'scan_id', 'scan_type'], name='scan_unique_constraint'
-            )
-        ]
-        ordering = ['scan_id']
+        ordering = ['name', 'scan_type']
```

Comment:
Can different timesteps have different `scan_type`? If not, why order by it? If yes, I assume it is only due to convenience, despite us not expecting it?

Reply:
This ordering is set on the `Scan` model, not the `Image` model. We do not expect Images/Frames to have different scan types, but different scans in the same experiment (possibly even with the same name) can have different scan types.

Reply:
Ok, changing to just order by name.